src/main/java/hudson/plugins/emailext/plugins/recipients/CulpritsRecipientProvider.java

```diff
@@ -51,10 +54,18 @@ public void send(final String format, final Object... args) {
         }
         final Debug debug = new Debug();
         Run<?,?> run = context.getRun();
-        if (run instanceof AbstractBuild) {
-            Set<User> users = ((AbstractBuild<?,?>)run).getCulprits();
-            RecipientProviderUtilities.addUsers(users, context, env, to, cc, bcc, debug);
-        } else {
+        // TODO: core 2.60+, workflow-job 2.12+: Switch to checking if run is RunWithSCM and make catch an else block
```

Comment:
Actually, I like what I've got here. It's a nice better-safe-than-sorry fallback.